src/main/java/dev/lambdaurora/lambdynlights/LambDynLightsCompat.java

```diff
@@ -59,4 +59,14 @@ public static boolean isSodium05XInstalled() {
 			}
 		}).orElse(false);
 	}
+
+	public static boolean isFabricApi086OrLaterInstalled() {
+		return FabricLoader.getInstance().getModContainer("fabric-api").map(mod -> {
```

Comment:
This is not a good way to check if the fix needs to be applied, the mod should check for the Indigo module directly as some mod could bundle any Fabric API modules without requiring the whole of it.